docs/components/table-view.md

```diff
@@ -0,0 +1,1016 @@
+# Table View
+
+Component for displaying data in table format.
+
+:::tip NOTE
+KTableView is a display-only component, it does not provide any data management capabilities like fetching, pagination, sorting or searching. If you are looking for a component that includes those functions out of the box, check out [KTable](/components/table).
+:::
+
+<KTableView
+  :data="basicData"
+  :headers="basicHeaders()"
+/>
+
+```html
+<KTableView
+  :data="tableData"
+  :headers="headers"
+/>
+```
+
+## Props
+
+### headers
+
+Array of objects that represent table columns along with some configurations that should apply to each of the columns (whether a column is sortable, hidable, etc).
+
+```ts
+interface TableViewHeader {
+  key: string | 'actions' // must be unique for each column, see Reserved Header Keys section for more information about 'actions' key value
```

Comment:
issue: I'm not sure `actions` should be in the interface here; see my other comment.